### PR TITLE
Use occ to install ownCloud in autotest.sh

### PR DESCRIPTION
**Pull request by DeepDiver1975**
Errors during setup will now be printed to the teriminal

``` sh
 (use-occ-setup-in-autotest-master) ~/Development/ownCloud/master$ ./autotest.sh sqlite
Using database oc_autotest
Setup environment for sqlite testing ...
Installing ....
PHP module GD not installed.
Please ask your server administrator to install the module.

PHP modules have been installed, but they are still listed as missing?
Please ask your server administrator to restart the web server.

Writing to database failedTesting with sqlite ...
PHP Fatal error:  Uncaught exception 'Exception' with message 'Not installed' in /home/deepdiver/Development/ownCloud/master/lib/base.php:241
Stack trace:
#0 /home/deepdiver/Development/ownCloud/master/lib/base.php(543): OC::checkInstalled()
#1 /home/deepdiver/Development/ownCloud/master/lib/base.php(1030): OC::init()
#2 /home/deepdiver/Development/ownCloud/master/tests/bootstrap.php(16): require_once('/home/deepdiver...')
#3 phar:///usr/local/bin/phpunit/phpunit/Util/Fileloader.php(58): include_once('/home/deepdiver...')
#4 phar:///usr/local/bin/phpunit/phpunit/Util/Fileloader.php(42): PHPUnit_Util_Fileloader::load('/home/deepdiver...')
#5 phar:///usr/local/bin/phpunit/phpunit/TextUI/Command.php(792): PHPUnit_Util_Fileloader::checkAndLoad('/home/deepdiver...')
#6 phar:///usr/local/bin/phpunit/phpunit/TextUI/Command.php(622): PHPUnit_TextUI_Command->handleBootstrap('/home/deepdiver...')
#7 phar:///usr/local/bin/phpunit/phpunit/TextUI/Command.php(114): PHPUnit_TextUI_Command->handleArguments(Array)
#8 phar:///usr/local/bin/phpu in /home/deepdiver/Development/ownCloud/master/lib/base.php on line 241
```

### CODE REVIEW

File: autotest.sh [hunk 1]
```diff
@@ -196,15 +133,13 @@ EOF
...
 
-	# copy autoconfig
-	cp "$BASEDIR/tests/autoconfig-$1.php" "$BASEDIR/config/autoconfig.php"
-
 	# trigger installation
-	echo "INDEX"
-	php -f index.php | grep -i -C9999 error && echo "Error during setup" && exit 101
-	echo "END INDEX"
+    echo "Installing ...."
+    ./occ maintenance:install --database=$1 --database-name=$DATABASENAME --database-host=localhost --database-user=$DATABASENAME --database-pass=owncloud --database-table-prefix=oc_ --admin-user=$ADMINLOGIN --admin-pass=admin --data-dir=$DATADIR
```
Comment (MorrisJobke) [inline autotest.sh]: Intendation

**Comment by MorrisJobke** [comment]
Oh yes :) I planned to do this too :) Nice move

**Comment by karlitschek** [comment]
Very helpful! :+1:

File: autotest.sh [hunk 2]
```diff
@@ -196,15 +133,13 @@ EOF
 			to $DATABASENAME;
 			exit;
 EOF
+        DATABASEUSER=$DATABASENAME
+        DATABASENAME='XE'
```
Comment (Xenopathic) [inline autotest.sh]: Whitespace errors here

**Comment by Xenopathic** [comment]
:+1: once those whitespace errors are fixed. We should probably work on merging the multiple autotest.sh files into one, with command line options to change the behaviour perhaps

**Comment by Xenopathic** [comment]
Non-static Setup! :clap:

**Comment by MorrisJobke** [comment]
Setup still works and code looks good :+1:

**Comment by DeepDiver1975** [comment]
> Setup still works and code looks good

let me squash these commits ....

**Pull request merged**